MeshLib/VtkOGSEnum.cpp

```diff
@@ -92,7 +92,7 @@ int OGSToVtkCellType(CellType ogs)
 			return VTK_PYRAMID;
 		case CellType::PYRAMID13:
 			return VTK_QUADRATIC_PYRAMID;
-		case CellType::INVALID:
+		default:
```

Comment:
This is reverting the commit 65e52ab9b which was intended to explicitly handle all possible enum types in the switch case.
If one would add an entry to the CellType enum a warning about not all handled cases would be shown.

Now adding a default case makes it possible to add another CellType entry without a warning.